src/tests/ftest/deployment/ior_per_rank.py

```diff
@@ -0,0 +1,130 @@
+            self.ior_cmd.transfer_size.update(self.transfer_sizes[idx])
+            dfs_out = self.run_ior_with_pool(fail_on_warning=self.log.info)
+            dfs_perf = IorCommand.get_ior_metrics(dfs_out)
+
+            # Destroy container, to be sure we use newly created container in next iteration
+            self.container.destroy()
+            self.container = None
+
+            # gather actual and expected perf data to be compared
+            if idx == 0:
+                dfs_max_write = float(dfs_perf[0][IorMetrics.Max_MiB])
+                dfs_max_read = float(dfs_perf[1][IorMetrics.Max_MiB])
+                actual_write_x = percent_change(dfs_max_write, self.expected_bw)
+                actual_read_x = percent_change(dfs_max_read, self.expected_bw)
+            else:
+                dfs_max_write = float(dfs_perf[0][IorMetrics.Max_OPs])
+                dfs_max_read = float(dfs_perf[1][IorMetrics.Max_OPs])
+                actual_write_x = percent_change(dfs_max_write, self.expected_iops)
+                actual_read_x = percent_change(dfs_max_read, self.expected_iops)
+
+            # compare actual and expected perf data
+            try:
+                self.assertLessEqual(abs(actual_write_x), self.write_x,
+                                     "Max Write Diff too large for rank: {}".format(rank))
+                self.assertLessEqual(abs(actual_read_x), self.read_x,
+                                     "Max Read Diff too large for rank: {}".format(rank))
+                # collect list of good nodes
+                good_node = self.server_managers[0].get_host(rank)
+                if good_node not in self.good_nodes:
+                    self.good_nodes.append(good_node)
+            except TestFail as _error:
+                # collect bad nodes
+                failed_node = self.server_managers[0].get_host(rank)
+                if failed_node not in self.failed_nodes:
+                    self.failed_nodes[failed_node] = [rank]
+                else:
+                    self.failed_nodes[failed_node].append(rank)
+                if failed_node in self.good_nodes:
+                    self.good_nodes.remove(failed_node)
+        # Destroy pool, to be sure we use newly created pool in next iteration
+        self.pool.destroy()
+        self.pool = None
+
+    def test_iorperrank(self):
+        """
+        Test Description: Test to check node health using ior with daos.
+                          Start entire daos system on each rack of a group.
+                          Create pool for each rank.
+                          Run ior using 4 good clients over each pool.
+                              Large transfer size: 1M
+                              Small transfer size: 256B
+                          Compare results and isolate bad nodes.
+        :avocado: tags=all,deployment,full_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=ior
+        :avocado: tags=iorperrank
```

Comment:
Feature tag and method name as the unique tag
```suggestion
        :avocado: tags=ior,deployment
        :avocado: tags=iorperrank,test_ior_per_rank
```